cache_registry/sync/auditors.py

```diff
@@ -0,0 +1,70 @@
+from flask import current_app
+
+from cache_registry.models import db, Auditor, User
+from cache_registry.sync import parsers
+from cache_registry.sync.auth import patch_users
+from cache_registry.sync.bdr import get_absolute_url
+from cache_registry.sync.utils import (
+    get_logger,
+    get_response,
+    update_contact_persons,
+)
+
+
+def get_latest_auditors(updated_since=None, page_size=None, uid=""):
+    """Get latest auditors from specific API url"""
+
+    url = get_absolute_url("API_URL_FGAS", "/latest/auditor/")
+    params = {}
+
+    if uid:
+        return [get_response(f"{url}{uid}", params)]
+
+    if updated_since:
+        params["updatedSince"] = updated_since.strftime("%d/%m/%Y")
+
+    if page_size:
+        params["pageSize"] = page_size
+        params["pageNumber"] = 1
+
+    return get_response(url, params)
+
+
+def add_updates_log(auditor, data):
+    existing_auditor_data = f"""
+        Auditor UID: {auditor.auditor_uid}
+        Auditor Name: {auditor.name}
+        Auditor Status: {auditor.status}
+        Auditor date_created: {auditor.date_created}
+        Auditor date_updated: {auditor.date_updated}
+        Auditor date_created_in_ecr: {auditor.date_created_in_ecr}
+        Auditor date_updated_in_ecr: {auditor.date_updated_in_ecr}
+    """
+    received_auditor_data = f"Received data: {data}"
+    message = f"""
+        Auditor {auditor.auditor_uid} updated: \n \
+        Old data: {existing_auditor_data} New data: {received_auditor_data}"
+    """
+    logger = get_logger("updated")
+    logger.info(message)
+
+
+def update_auditor(data):
+    """Create or update auditor from received data"""
+
+    original_data = data.copy()
+    contact_persons = parsers.parse_cp_list(data.pop("contactPersons", []))
+    contact_persons, _ = patch_users(data["auditorUID"], contact_persons)
```

Comment:
As I mentioned in the localsettings.py.example, be careful here as a different dictionary should be used for patching users from auditors